content/components/selectpanel.mdx

```diff
@@ -0,0 +1,222 @@
+| <Box as="kbd" backgroundColor="canvas.subtle" borderRadius={1} borderColor="border.default" borderWidth={1} borderStyle="solid" sx={{ whiteSpace: "nowrap"}} py={1} px={2}>Shift + Tab</Box> | Move focus backward between interactive controls and widgets within the dialog.                                             |
+| <Box as="kbd" backgroundColor="canvas.subtle" borderRadius={1} borderColor="border.default" borderWidth={1} borderStyle="solid" py={1} px={2}>ArrowDown</Box>                                | Move focus from the input to the listbox. (Shift+Tab required to move from listbox to input)                                |
+| <Box as="kbd" backgroundColor="canvas.subtle" borderRadius={1} borderColor="border.default" borderWidth={1} borderStyle="solid" py={1} px={2}>Home</Box>                                     | Move focus to first item in listbox.                                                                                        |
+| <Box as="kbd" backgroundColor="canvas.subtle" borderRadius={1} borderColor="border.default" borderWidth={1} borderStyle="solid" py={1} px={2}>PageUp</Box>                                   | Move focus to first item in listbox.                                                                                        |
+| <Box as="kbd" backgroundColor="canvas.subtle" borderRadius={1} borderColor="border.default" borderWidth={1} borderStyle="solid" py={1} px={2}>End</Box>                                      | Move focus to last item in listbox.                                                                                         |
+| <Box as="kbd" backgroundColor="canvas.subtle" borderRadius={1} borderColor="border.default" borderWidth={1} borderStyle="solid" py={1} px={2}>PageDown</Box>                                 | Move focus to last item in listbox.                                                                                         |
+| <Box as="kbd" backgroundColor="canvas.subtle" borderRadius={1} borderColor="border.default" borderWidth={1} borderStyle="solid" py={1} px={2}>Esc</Box>                                      | Close dialog and clear any changed inputs.                                                                                  |
+
+### Multi-select
+
+| Key                                                                                                                                                           | Description                         |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------- |
+| <Box as="kbd" backgroundColor="canvas.subtle" borderRadius={1} borderColor="border.default" borderWidth={1} borderStyle="solid" py={1} px={2}>ArrowDown</Box> | Navigate through the listbox items. |
```

Comment:
Is it worth adding `ArrowUp` here too, or do we omit that as convention?

Reply:
I could add ArrowUp for posterity. If you're at the first item in the listbox, nothing will happen, but if you're in any other item in the listbox, it will move to the item previous.

Reply:
I'm trying to keep the Multi-select and Single-select sections specific to just their different behaviors and any shared behaviors to be within Select panel. If that's confusing I could just remove that Select panel section and duplicate info